.github/workflows/build.yml

```diff
@@ -16,7 +12,6 @@ jobs:
     runs-on: windows-latest
     env:
       Solution_Name: Acrolinx.Demo.Sidebar.Net.sln
-      NUGET_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
NUGET_TOKEN variable is needed to download SDK artifact dependencies from Github. It is used by Visual Studio while restoring nuget packages

Reply:
Added NUGET_TOKEN again